src/js/actions/search/menucommands.js

```diff
@@ -53,7 +53,13 @@ define(function (require, exports) {
             }
 
             if (labels[part].$MENU) {
-                path += labels[part].$MENU + ">";
+                // Don't show "Application" options on Windows
```

Comment:
Which is to say that it seems like the one-line diff to the strings file is all that should be needed?